calculix + exodus (from seacas package) error with static xml2

I have this error for `$(shell pkg-config --libs --static --cflags libxml-2.0)` in Makefile:

```
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/lib\libnetcdf.a(ncxml_xml2.c.obj):(.text+0x13): undefined reference to `__imp_xmlCleanupParser'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/lib\libnetcdf.a(ncxml_xml2.c.obj):(.text+0x38): undefined reference to `__imp_xmlReadMemory'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/lib\libnetcdf.a(ncxml_xml2.c.obj):(.text+0x53): undefined reference to `__imp_xmlFreeDoc'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/lib\libnetcdf.a(ncxml_xml2.c.obj):(.text+0x63): undefined reference to `__imp_xmlDocGetRootElement'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/lib\libnetcdf.a(ncxml_xml2.c.obj):(.text+0x9a): undefined reference to `__imp_xmlGetProp'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe:CC:/msys64/mingw64/lib\libnetcdf.a(ncxml_xml2.c.obj):(.text+0xb4): undefined reference to `__imp_xmlFree'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/lib\libnetcdf.a(ncxml_xml2.c.obj):(.text+0x17d): undefined reference to `__imp_xmlNodeGetContent'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/lib\libnetcdf.a(ncxml_xml2.c.obj):(.text+0x199): undefined reference to `__imp_xmlFree'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/lib\libnetcdf.a(ncxml_xml2.c.obj):(.text+0x289): undefined reference to `__imp_xmlNodeListGetString'
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: C:/msys64/mingw64/lib\libnetcdf.a(ncxml_xml2.c.obj):(.text+0x290): undefined reference to `__imp_xmlFree'
collect2.exe: error: ld returned 1 exit status
```

but if I replace it by `/mingw64/lib/libxml2.dll.a` this error no longer occurs, however the program requires dynamic libraries, my goal is a fully static exe file (about 2 - 3 years ago I had a completely static version: https://calculix.discourse.group/t/calculix-and-pastix-solver-windows-version/130/51?u=rafal.brzegowy)

Required dll libraries:
```
libiconv-2.dll
liblzma-5.dll
libxml2-2.dll
zlib1.dll
```

Source files: https://github.com/gustafson/CalculiX/tree/exodus

--- a/mingw-w64-netcdf/PKGBUILD
+++ b/mingw-w64-netcdf/PKGBUILD
@@ -4,7 +4,7 @@ _realname=netcdf
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 pkgver=4.9.2
-pkgrel=1
+pkgrel=2
 pkgdesc="Interface for scientific data access to large binary data (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64' 'mingw32' 'clang32')
@@ -40,6 +40,10 @@ prepare() {
 }
 
 build() {
+  local -a _static_flags=(
+    -DLIBXML_STATIC
+  )
+
   declare -a extra_config
   if check_option "debug" "n"; then
     extra_config+=("-DCMAKE_BUILD_TYPE=Release")
@@ -55,6 +59,8 @@ build() {
 
   # Disable BYTERANGE because that feature doesn't build with HDF5 1.14 in netCDF 4.9.1
   # It was disabled by default in netCDF 4.9.0
+  CFLAGS+=" ${_static_flags[@]}" \
+  CXXFLAGS+=" ${_static_flags[@]}" \
   MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
   "${MINGW_PREFIX}"/bin/cmake.exe \
     -G"MSYS Makefiles" \

